application-diagram-ui/src/main/resources/Diagram/DiagramEditSheet.xml

```diff
@@ -1051,6 +1051,44 @@ define('diagram-url-io', ['diagram-utils', 'draw.io'], function(diagramUtils) {
   }
 });
 
+/**
+  * In case external services are disabled, stop the features that require it (export of pdf from diagram editor
+  * or png / jpeg export on IE) and show an info dialog.
+  */
+define('diagram-external-services', ['jquery', 'diagram-config', 'draw.io'], function($, diagramConfig) {
+  var showDisabledServicesDialog = function(editorUi) {
+    var errorMessage = $('&lt;div&gt;&lt;/div')
+      .html("$services.localization.render('diagram.editor.disabledExternalServices')");
```

Comment:
You also need to do JavaScript escaping (e.g. if the translation message contains a new line or a quote). I usually do it like this:

    .html($jsontool.serialize($services.localization.render('diagram.editor.disabledExternalServices')));